src/images/svg/symbols/gbp-sgd.svg

```diff
@@ -1,27 +1,23 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="33" viewBox="0 0 32 33">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
     <defs>
-        <rect id="gbpsgdprefix__a" width="24" height="16" x="0" y="0" rx="2"/>
-        <rect id="gbpsgdprefix__c" width="24" height="16" x="0" y="0" rx="2"/>
+        <rect id="prefix__a" width="24" height="16" x="0" y="0" rx="2"/>
     </defs>
-    <g fill="none" fill-rule="evenodd" transform="translate(0 5)">
+    <g fill="none" fill-rule="evenodd" transform="translate(0 4)">
         <g>
-            <mask id="gbpsgdprefix__b" fill="#fff">
-                <use xlink:href="#gbpsgdprefix__a"/>
```

Comment:
same here, we should use the related id for the `xlink:href`
it's for preventing the id replication.